genesyscloud/resource_genesyscloud_telephony_providers_edges_phone_test.go

```diff
@@ -182,44 +181,24 @@ func TestAccResourcePhoneBasic(t *testing.T) {
 	})
 }
 
-func deleteDidPoolWithNumber(number string) error {
-	//sdkConfig := m.(*ProviderMeta).ClientConfig
-	edgesAPI := platformclientv2.NewTelephonyProvidersEdgeApiWithConfig(sdkConfig)
-
-	for pageNum := 1; ; pageNum++ {
-		const pageSize = 100
-		didPools, _, getErr := edgesAPI.GetTelephonyProvidersEdgesDidpools(pageSize, pageNum, "", nil)
-		if getErr != nil {
-			return getErr
-		}
-
-		if didPools.Entities == nil || len(*didPools.Entities) == 0 {
-			break
-		}
-
-		for _, didPool := range *didPools.Entities {
-			if (didPool.StartPhoneNumber != nil && *didPool.StartPhoneNumber == number) ||
-				(didPool.EndPhoneNumber != nil && *didPool.EndPhoneNumber == number) {
-				if _, err := edgesAPI.DeleteTelephonyProvidersEdgesDidpool(*didPool.Id); err != nil {
-					return err
-				}
-				time.Sleep(5 * time.Second)
-			}
-		}
-	}
-	return nil
-}
-
 func TestAccResourcePhoneStandalone(t *testing.T) {
 	t.Parallel()
-	didPoolResource1 := "test-didpool1"
 	number := "+14175538114"
-	if _, err := AuthorizeSdk(); err != nil {
+	platformConfig, err := AuthorizeSdk()
+	if err != nil {
 		t.Fatal(err)
 	}
-	if err := deleteDidPoolWithNumber(number); err != nil {
+	// TODO: Use did pool resource inside config once cyclic dependency issue is resolved between genesyscloud and did_pools package
+	didPoolId, err := createDidPoolForEdgesPhoneTest(platformConfig, number)
+	if err != nil {
 		t.Fatal(err)
 	}
+	defer func() {
+		if err := deleteDidPool(platformConfig, didPoolId); err != nil {
+			t.Logf("failed to delete did pool '%s': %v", didPoolId, err)
+		}
+	}()
```

Comment:
Same situation here as with the emergency group tests. DID Pool resource can't be registered for these tests, so these functions create and delete the DID pool